packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/README.md

```diff
@@ -19,8 +19,9 @@
 15. [Patrol integration tests](#patrol-integration-tests){{/enable_patrol}}{{#realtime_communication}}
 16. [Realtime communication](#realtime-communication){{/realtime_communication}}{{#enable_feature_otp}}
 17. [Feature OTP](#feature-otp){{/enable_feature_otp}} {{#cicd}}
-18. [CI/CD](#cicd){{/cicd}}
-19. [Next Steps](#next-steps)
+18. [CI/CD](#cicd){{/cicd}} {{#enable_pin_code}}
+19. [Pin Code](#pin-code){{/enable_pin_code}}
```

Comment:
The link won't take us to the correct location

Reply:
The link is fixed

Reply:
The link is fixed